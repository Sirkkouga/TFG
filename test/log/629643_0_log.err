INFO - 06/04/24 15:51:23 - 0:00:00 - Read 1071 words from the vocabulary file.

INFO - 06/04/24 15:51:23 - 0:00:00 - 870 words (1071 unique) in 9 sentences.
INFO - 06/04/24 15:51:23 - 0:00:00 - 17 unknown words (5 unique), covering 1.95% of the data.
INFO - 06/04/24 15:51:23 - 0:00:00 - jav@@: 6
INFO - 06/04/24 15:51:23 - 0:00:00 - ache: 5
INFO - 06/04/24 15:51:23 - 0:00:00 - pression: 3
INFO - 06/04/24 15:51:23 - 0:00:00 - Over@@: 2
INFO - 06/04/24 15:51:23 - 0:00:00 - :@@: 1
INFO - 06/04/24 15:51:23 - 0:00:00 - Job completed successfully
INFO - 06/04/24 15:51:23 - 0:00:00 - Exiting after successful completion
