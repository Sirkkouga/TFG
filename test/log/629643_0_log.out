submitit INFO (2024-06-04 15:51:20,288) - Starting with JobEnvironment(job_id=629643, hostname=veuc09, local_rank=0(1), node=0(1), global_rank=0(1))
submitit INFO (2024-06-04 15:51:20,289) - Loading pickle: /home/usuaris/veu/javier.abella/TFG/test/log/629643_submitted.pkl
Saving the data to /home/usuaris/veu/javier.abella/TFG/test/java.valid.sa.bpe.pth ...
submitit INFO (2024-06-04 15:51:23,669) - Job completed successfully
submitit INFO (2024-06-04 15:51:23,671) - Exiting after successful completion
